django/cantusdb_project/main_app/views/chant.py

```diff
@@ -67,6 +67,139 @@
+    },
+    {
+        "name": "Portuguese Early Music Database",
+        "initialism": "PEM",
+        "base_url": "https://pemdatabase.eu",
+        "results_fstring": "https://pemdatabase.eu/id/{cantus_id}",
+    },
+    {
+        "name": "Spanish Early Music Manuscript Database",
+        "initialism": "SEMM",
+        "base_url": "http://musicahispanica.eu",
+        "results_fstring": "http://musicahispanica.eu/id/{cantus_id}",
+    },
+    {
+        "name": "Cantus Planus in Polonia",
+        "initialism": "CPL",
+        "base_url": "http://cantus.ispan.pl",
+        "results_fstring": "http://cantus.ispan.pl/id/{cantus_id}",
+    },
+    {
+        "name": "Hungarian Chant Database",
+        "initialism": "HCD",
+        "base_url": "http://hun-chant.eu",
+        "results_fstring": "http://hun-chant.eu/id/{cantus_id}",
+    },
+    {
+        "name": "Cantus Planus in Slovakia",
+        "initialism": "CSK",
+        "base_url": "http://cantus.sk",
+        "results_fstring": "http://cantus.sk/id/{cantus_id}",
+    },
+    {
+        "name": "Fragmenta Manuscriptorum Musicalium Hungariae",
+        "initialism": "FRH",
+        "base_url": "http://fragmenta.zti.hu/en/",
+        "results_fstring": "http://fragmenta.zti.hu/en/id/{cantus_id}",
+    },
+)
+
+
+def make_concordances_database_summary(
+    concordances: List[dict],
+    cantus_id: str,
+) -> List[dict]:
+    """Create list of dictionaries containing data to
+    be unpacked in chant_detail.html to make concordances
+    summary.
+    i.e. this summary:
+        Displaying N concordances from the following databases:
+        Cantus Database (CD) - N chants
+        Fontes Cantus Bohemiae (FCB) - N chants
+        etc...
+        Gregorien.info - VIEW AT GREGORIEN.INFO
+
+    Args:
+        concordances (List[dict]): list of concordant chants, provided
+            by the json-cid API on Cantus Index
+        cantus_id (str): _description_
+
+    Returns:
+        List[dict]: data on how many concordant chants are found in which
+            databases. Each item in the list is a dictionary corresponding
+            to a single database in the Cantus Network.
+    """
+    database_summaries: List[dict] = []
+    for database in CANTUS_NETWORK_DATABASES:
+        name: str = database["name"]
+        initialism: str = database["initialism"]
+        base_url: str = database["base_url"]
+        results_url_template: str = database["results_fstring"]
+        results_url: str = results_url_template.format(cantus_id=cantus_id)
+        results_count: int = len([True for c in concordances if c["db"] == initialism])
+        database_summary: dict = {
+            "name": name,
+            "initialism": initialism,
+            "base_url": base_url,
+            "results_url": results_url,
+            "results_count": results_count,
+        }
+        database_summaries.append(database_summary)
+
+    filtered_database_summaries: List[dict] = [
```

Comment:
Maybe clearer to put this conditional before you append to `database_summaries` in the `for` block above.